app/src/layouts/kanban/kanban.vue

```diff
@@ -111,16 +120,17 @@ export default {
 import { addTokenToURL } from '@/api';
 import { getItemRoute } from '@/utils/get-item-route';
 import { getRootPath } from '@/utils/get-root-path';
-import { ref } from 'vue';
+import { ref, computed } from 'vue';
 import { useI18n } from 'vue-i18n';
 import Draggable from 'vuedraggable';
-import type { ChangeEvent, Group, Item } from './types';
+import type { ChangeEvent, Group, Item, LayoutOptions } from './types';
+import type { Field } from '@directus/types';
 
 const props = withDefaults(
 	defineProps<{
 		collection?: string | null;
 		groupCollection?: string | null;
-		fieldsInCollection?: Record<string, any>[];
+		fieldsInCollection?: Field[];
```

Comment:
It seems that when `fieldsInCollection` is passed to the Kanban layout, it is a `Field[]`. I updated the type here to reflect the actual objects passed to the layout.